package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "standard",
     "prebuild": "rimraf dist",
     "prepublish": "npm run build",
-    "test": "mocha --compilers js:babel-register 'src/**/*.test.js'"
+    "test": "mocha --compilers js:babel-register \"src/**/*.test.js\""
```

Comment:
(Fix test pattern for Windows users, see mochajs/mocha#2895 )

Reply:
Nice!